pandas/core/generic.py

```diff
@@ -3583,8 +3583,8 @@ def replace(self, to_replace=None, value=None, inplace=False, limit=None,
               wrappers around the scipy interpolation methods of similar
               names. These use the actual numerical values of the index. See
               the scipy documentation for more on their behavior
-              `here <http://docs.scipy.org/doc/scipy/reference/interpolate.html#univariate-interpolation>`__  # noqa
-              `and here <http://docs.scipy.org/doc/scipy/reference/tutorial/interpolate.html>`__  # noqa
```

Comment:
Content wise, it maybe also nicer instead of "See the scipy documention for more on their behaviour here and here" to read "For more on their behaviour see the scipy reference and tutorial documentation" (then the difference between the two 'here's is more clear)